src/tests/test_util.py

```diff
@@ -143,3 +175,375 @@ def test_parse_sql_results_to_json(ip, capsys, rows, columns, expected_json):
+            "sqlplot",
+            ["--table", "--column", "--breaks", "-B"],
+            [("B", "breaks")],
+        ),
+        # for binwidth/W
+        (
+            "sqlplot",
+            ["--table", "--column", "--binwidth", "--binwidth"],
+            [],
+        ),
+        (
+            "sqlplot",
+            ["--table", "--column", "-W", "-W"],
+            [],
+        ),
+        (
+            "sqlplot",
+            ["--table", "--column", "--binwidth", "-W"],
+            [("W", "binwidth")],
+        ),
+        # for orient/o
+        (
+            "sqlplot",
+            ["--table", "--column", "--orient", "--orient"],
+            [],
+        ),
+        (
+            "sqlplot",
+            ["--table", "--column", "-o", "-o"],
+            [],
+        ),
+        (
+            "sqlplot",
+            ["--table", "--column", "--orient", "-o"],
+            [("o", "orient")],
+        ),
+        # for show-numbers/S
+        (
+            "sqlplot",
+            ["--table", "--column", "--show-numbers", "--show-numbers"],
+            [],
+        ),
+        (
+            "sqlplot",
+            ["--table", "--column", "-S", "-S"],
+            [],
+        ),
+        (
+            "sqlplot",
+            ["--table", "--column", "--show-numbers", "-S"],
+            [("S", "show-numbers")],
+        ),
+        # for mixed
+        (
+            "sqlplot",
+            [
+                "--table",
+                "--column",
+                "--column",
+                "-w",
+                "--with",
+                "--show-numbers",
+                "--show-numbers",
+                "--binwidth",
+                "--orient",
+                "-o",
+                "--breaks",
+                "-B",
+            ],
+            [("o", "orient"), ("B", "breaks")],
+        ),
+        #
+        # FOR SQLCMD
+        #
+        # for schema/s
+        (
+            "sqlcmd",
+            ["--schema", "--schema"],
+            [],
+        ),
+        (
+            "sqlcmd",
+            ["-s", "-s"],
+            [],
+        ),
+        (
+            "sqlcmd",
+            ["--schema", "-s"],
+            [("s", "schema")],
+        ),
+        # for table/t
+        (
+            "sqlcmd",
+            ["--table", "--table"],
+            [],
+        ),
+        (
+            "sqlcmd",
+            ["-t", "-t"],
+            [],
+        ),
+        (
+            "sqlcmd",
+            ["--table", "-t"],
+            [("t", "table")],
+        ),
+        # for mixed
+        (
+            "sqlcmd",
+            ["--table", "-t", "-s", "-s", "--schema"],
+            [("t", "table"), ("s", "schema")],
+        ),
+    ],
+)
+def test_check_duplicate_arguments_raises_usageerror(
+    check_duplicate_message_factory,
+    cmd_from,
+    args,
+    aliases,
+):
+    with pytest.raises(UsageError) as excinfo:
+        util.check_duplicate_arguments(cmd_from, args)
+    print(str(excinfo.value))
```

Comment:
Looks like a print statement for debugging? If so should be removed

Reply:
My bad!

Reply:
Removed the print statement in this [commit](https://github.com/ploomber/jupysql/pull/908/commits/122607f267a48195c81dea3a95da50aa63dc32dc)